includes/Admin/Settings.php

```diff
@@ -73,6 +72,7 @@ public function add_menu_item() {
 
 		$root_menu_item       = 'woocommerce';
 		$is_marketing_enabled = false;
+		$this->use_woo_nav = class_exists( WooAdminFeatures::class ) && class_exists( WooAdminMenu::class ) && WooAdminFeatures::is_enabled( 'navigation' );
```

Comment:
Minor: even though `phpcs:ignore` is set for this file, we probably want to align the equals signs anyways.

Reply:
Thanks for looking at this. I totally missed this. I have corrected the alignment on the variable assignment.